subsys/spm/spm.c

```diff
@@ -160,6 +161,9 @@ extern int irq_target_state_is_secure(unsigned int irq);
 	((SPU_PERIPHID_PERM_LOCK_Locked << SPU_PERIPHID_PERM_LOCK_Pos) &       \
 	 SPU_PERIPHID_PERM_LOCK_Msk)
 
+#define LAST_SECURE_ADDRESS (PM_CFG_SPM_ADDRESS + PM_CFG_SPM_SIZE)
+#define LAST_SECURE_REGION_INDEX ((LAST_SECURE_ADDRESS/(32*1024)) - 1)
```

Comment:
Please add a define, and possibly a comment for the 32kB.

Reply:
See change for proposal. 

Alternative would be to add a node in DTS, but AFAICS that could be a future improvement to save time.

Reply:
LGTM